examples/asr/transcribe_speech.py

```diff
@@ -99,6 +99,9 @@ class TranscriptionConfig:
     # Decoding strategy for RNNT models
     rnnt_decoding: RNNTDecodingConfig = RNNTDecodingConfig(fused_batch_size=-1)
 
+    # Set enc params which may have been set randomly during training
+    val_random_params_enc: Dict[str, Any] = field(default_factory=dict)
```

Comment:
This is not good design.  It specialises to exactly one model. Please remove from the general transcribe speech api.